### PR TITLE
Add gpt3 175b on v5e config

### PR DESCRIPTION
**Pull request by raymondzouu**
Adding GPT-3 175B training config on v5e.

### CODE REVIEW

File: MaxText/configs/v5e/gpt3_175b.sh [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+if [ -n "$RUN_NAME" ];
+then
+    export M_RUN_NAME=$RUN_NAME
+fi
+
+# Set up network
+bash preflight.sh PLATFORM=$PLATFORM
+
+# Train
+export LIBTPU_INIT_ARGS="--xla_tpu_enable_data_parallel_all_reduce_opt=true --xla_tpu_data_parallel_opt_different_sized_ops=true --xla_tpu_enable_async_collective_fusion=true --xla_tpu_enable_async_collective_fusion_fuse_all_gather=true --xla_tpu_enable_async_collective_fusion_multiple_steps=true --xla_tpu_overlap_compute_collective_tc=true --xla_enable_async_all_gather=true --xla_tpu_spmd_rng_bit_generator_unsafe=true"
```
Comment (ZhiyuLi-goog) [inline MaxText/configs/v5e/gpt3_175b.sh:41]: Hi, @raymondzouu, `--xla_tpu_spmd_rng_bit_generator_unsafe=true` is necessary for int8 quantization only. We can remove it without int8 quantization. Then the `LIBTPU_INIT_ARGS` is the same as the other recipe in under v5e folder.

**Comment by ZhiyuLi-goog** [comment]
Pretty cool! Thank you @raymondzouu ! LGTM except the nit.

**Review by ZhiyuLi-goog — APPROVED** [review]

File: MaxText/configs/v5e/gpt3_175b.sh (hunk 1 above)
Comment (raymondzouu) [reply MaxText/configs/v5e/gpt3_175b.sh:41]: I see, thanks for the heads up. I am planning to test with and without int8 quantization in the XLML test by setting the env variable `export M_QUANTIZATION=int8`. Can we still leave in this flag when running without int8 quantization?

Comment (ZhiyuLi-goog) [reply MaxText/configs/v5e/gpt3_175b.sh:41]: Agreed!

**Review by gobbleturk — APPROVED** [review]

**Pull request merged**